math/armstrong_number/Python/armstrong_no.py

```diff
@@ -0,0 +1,12 @@
+n = int(input("Enter the number: "))
+arm_no = 0
+m = n 
+while(n!=0):
+    r = int(n % 10)
+    arm_no = arm_no + (r*r*r)
+    n = n / 10
+print(arm_no)
+if(m == arm_no):
+    print(m,"is a armstong number")
+else:
+    print(m,"is not a armstong number")
```

Comment:
Interesting concept! I did not know about that, thanks! 😄 

> ![image](https://user-images.githubusercontent.com/12429434/66873297-7c30c700-ef7e-11e9-85df-508ee83cf1e9.png)

From the above however, I can tell your code only works for 3-digit narcissistic numbers. Here's a (tested) suggestion on how to make that solution general for any sequence of digits:

```python
# A number X with N digits is an armstrong number if:
#   X == sum(Xi^N), for every digit Xi of X

# parse input
digits = str(input("Enter the number: "))
n_digits = len(digits)
target_number = int(digits)

# sum of exp's
sum_of_exps = sum([int(it) ** n_digits for it in digits])

# check result
if target_number == sum_of_exps:
    print(target_number, "is an armstong number")
else:
    print(target_number, "is NOT an armstong number")
```